tests/zndraw/test_share_token_dep.py

```
@@ -0,0 +1,75 @@
+
+from zndraw.access import ShareAccess, ShareContext
+from zndraw.dependencies import resolve_share_token
+from zndraw.models import RoomShareLink
+
+
+@pytest.mark.asyncio
+async def test_resolve_valid_token(session: AsyncSession) -> None:
+    room_id = "room-x"
+    uid = uuid4()
+    link = RoomShareLink(
+        room_id=room_id, token="tkn-1", access=ShareAccess.EDIT, created_by_id=uid
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "token"

(S106)